indico/modules/events/editing/schemas.py

```diff
@@ -162,3 +162,24 @@ def _check_for_unique_tag_code(self, code):
             query = query.filter(EditingTag.id != tag.id)
         if query.has_rows():
             raise ValidationError(_('Tag code must be unique'))
+
+
+class EditableFileTypeArgs(mm.Schema):
+    class Meta:
+        rh_context = ('event', 'file_type')
+
+    name = fields.String(required=True, validate=not_empty)
+    extension = fields.List(fields.String(), required=True)
+    allow_multiple_files = fields.Boolean(required=True)
+    required = fields.Boolean(required=True)
+    publishable = fields.Boolean(required=True)
```

Comment:
`extension` instead of `extensions`, and the list items may not be empty.

Also, with the changes below this schema will make the API more flexible once it's also used for creating new file types (since only the name will be required then).

```python
    extensions = fields.List(fields.String(validate=not_empty))
    allow_multiple_files = fields.Boolean()
    required = fields.Boolean()
    publishable = fields.Boolean()
```

Reply:
okay, got it :+1: